requirements.txt

```diff
@@ -1,5 +1,16 @@
+asgiref==3.7.2
+certifi==2023.7.22
+charset-normalizer==3.2.0
 flake8==5.0.4
-flake8-annotations==2.9.1
 flake8-quotes==3.3.1
 flake8-variables-names==0.0.5
+idna==3.4
+mccabe==0.7.0
 pep8-naming==0.13.2
+pycodestyle==2.9.1
+pyflakes==2.5.0
+pytz==2023.3
+requests==2.31.0
+sqlparse==0.4.4
+tzdata==2023.3
+urllib3==2.0.4
```

Comment:
also, why you need all this requirements?
if you use only `requests`?

Reply:
I myself don’t understand where they all came from(( I accidentally added Django, because I opened the wrong project. How the others were added, I don’t understand((